### PR TITLE
Update audit as of 2020-04-03

### PR DESCRIPTION
**Pull request by spiffxp**
I would like to squash this before merge, but I've split it up into individual commits for ease of review.

I tried to figure out what caused all of these changes, I think I have a handle on most of it, but would like another pair of eyes, especially on the k8s-nodes@kubernetes-public commit

### CODE REVIEW

File: audit/projects/k8s-artifacts-prod-bak/services/enabled.json [hunk 1]
```diff
@@ -91,6 +91,7 @@
       ],
       "requiredBy": [
         "appengineflex.googleapis.com",
+        "autopush-anthos.sandbox.googleapis.com",
```
Comment (listx) [inline audit/projects/k8s-artifacts-prod-bak/services/enabled.json:94]: I don't know who/what made these `*-anthos` changes but I assume they were don't automatically by some underlying system.

File: audit/projects/k8s-artifacts-prod/iam.json [hunk 2]
```diff
@@ -27,6 +27,12 @@
       ],
       "role": "roles/editor"
     },
+    {
+      "members": [
+        "group:k8s-infra-artifact-admins@kubernetes.io"
```
Comment (listx) [inline audit/projects/k8s-artifacts-prod/iam.json:32]: This was done by #707

File: audit/projects/k8s-artifacts-prod/service-accounts/k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com/iam.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "bindings": [
     {
       "members": [
-        "serviceAccount:k8s-prow-builds.svc.id.goog[test-pods/k8s-artifacts-prod]"
+        "serviceAccount:k8s-prow.svc.id.goog[test-pods/k8s-infra-gcr-promoter]"
```
Comment (listx) [inline audit/projects/k8s-artifacts-prod/service-accounts/k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com/iam.json:5]: This was done by #695

File: audit/projects/k8s-gcr-backup-test-prod-bak/service-accounts/k8s-infra-gcr-promoter@k8s-gcr-backup-test-prod-bak.iam.gserviceaccount.com/iam.json [hunk 4]
```diff
@@ -1 +1,11 @@
-{}
+{
+  "bindings": [
+    {
+      "members": [
+        "serviceAccount:k8s-prow-builds.svc.id.goog[test-pods/k8s-infra-gcr-promoter-test]"
```
Comment (listx) [inline audit/projects/k8s-gcr-backup-test-prod-bak/service-accounts/k8s-infra-gcr-promoter@k8s-gcr-backup-test-prod-bak.iam.gserviceaccount.com/iam.json:5]: This is from #710

**Comment by listx** [comment]
/lgtm

Aside: I would keep the commits un-squashed --- not sure why we want to squash it into one huge commit.

**Comment by spiffxp** [comment]
@listx fair point, I'll them unsquashed unless there is a strong desire to do otherwise

The reason these changes are large is that we're not running the audit script frequently enough.  Also, the commit I just added that _should_ represent only https://github.com/kubernetes/k8s.io/pull/714's changes looks like it represents more. So I'm not sure if we're running the ensure scripts completely or frequently enough.

Grouping into commits for this PR the way I did took time, it was how I was trying to make sense of the changes. I don't expect I'll have the bandwidth to do that too often.

**Comment by spiffxp** [comment]
I'm not 100% sure I'll be able to preserve the commits being broken up as they are after a rebase, I'll give it a shot though. Hoping https://github.com/kubernetes/k8s.io/pull/723 makes this a lot smaller

File: audit/projects/k8s-cip-test-prod/services/compute/project-info.json [hunk 5]
```diff
@@ -38,7 +38,7 @@
     {
       "limit": 250.0,
       "metric": "ROUTES",
-      "usage": 23.0
```
[the remark this answers is not in the document]
Comment (spiffxp) [reply audit/projects/k8s-cip-test-prod/services/compute/project-info.json:41]: Opened https://github.com/kubernetes/k8s.io/issues/726 for followup

**Comment by spiffxp** [comment]
rebased since https://github.com/kubernetes/k8s.io/pull/723 merged

File: audit/projects/k8s-artifacts-prod/buckets/artifacts.k8s-artifacts-prod.appspot.com/bucketpolicyonly.txt [hunk 6]
```diff
@@ -0,0 +1,3 @@
+Bucket Policy Only setting for gs://artifacts.k8s-artifacts-prod.appspot.com:
```
[the remark this answers is not in the document]
Comment (thockin) [reply audit/projects/k8s-artifacts-prod/buckets/artifacts.k8s-artifacts-prod.appspot.com/bucketpolicyonly.txt]: You should be able to delete this bucket's files entirely.

**Pull request merged**